internal/common/database/jobstates.go

```diff
@@ -0,0 +1,43 @@
+package database
+
+type JobState string
+
+type JobRunState string
+
+const (
```

Comment:
I'm not sure these can go in the `common/database` package because they are lookout specific.  I'm not entirely sure what the best thing to do is here though because they *are* common between ingester and lookout?  Maybe  put them in  `common/database/lookout`?